beta/adobe-journey-optimizer/push-messaging-guide.md

```diff
@@ -0,0 +1,81 @@
+
+This document will walk you through a basic push notifications end-to-end data flow using Adobe Journey Optimizer and a user flow diagram to explain how each role fulfills its responsibilities and collaborates to bring the push data flow together. 
+
+
+
+## Understanding Push Data Flow
+
+This diagram shows a basic push messaging data flow and gives a peek at the various Adobe products and components involved in the flow. 
+
+![Push Data Flow](../../.gitbook/assets/push-data-flow.png)
+
+
+
+### Push Data Flow Components
+
+#### Cloud Messaging Providers
+
+Cloud Messaging Providers are third-party services that enable us to deliver notifications from remote servers to mobile apps. We support both Android and iOS platforms and deal with two primary cloud messaging services :
+
+- [Firebase Cloud Messaging](https://firebase.google.com/docs/cloud-messaging) (FCM) - to send notifications to Android mobile app
+- [Apple Push Notification Service](https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/APNSOverview.html) (APNs) - to send notifications to iOS mobile app
+
+#### Mobile App Integrated with Adobe Mobile SDK
+
+Adobe Mobile SDK helps integrate your mobile apps with Adobe Experience Cloud Solutions like Experience Platform and Journey Optimizer. Mobile SDK is comprised of various Experience Cloud solution extensions to provide features specific to the service they represent. These extensions expose various [APIs](https://aep-sdks.gitbook.io/docs/beta/adobe-journey-optimizer/api-reference) to enable data flow like registering the push token or sending push tracking events or any other custom experience events to Adobe Experience Platform. 
+
+Mobile SDK has now integrated with [Project Griffon](https://aep-sdks.gitbook.io/docs/beta/project-griffon), which is a new, innovative beta product from Adobe Experience Cloud to help you inspect, proof, simulate, and validate how you collect data or serve experiences in your mobile app. With respect to push notifications, this integration helps validate various configurations across components involved in the data flow, by running a [test end-to-end push notification flow](https://aep-sdks.gitbook.io/docs/resources/troubleshooting-guides/troubleshooting-push). It shows errors encountered and recommendations to solve misconfigurations.
+
+#### Adobe Launch
+
+[Adobe Launch or Data Collection](https://experienceleague.adobe.com/docs/launch/using/home.html) is the next generation of mobile SDK management capabilities which enables data flow from mobile SDK to Experience Platform. It provides capabilities to register extensions, create rules and data elements to send data from your mobile app to the Adobe experience cloud solutions. With respect to the push notifications data flow, the primary configurations required in Launch are :
```

Comment:
"... enables data flow from Mobile SDK to Experience Platform."

Reply:
"... to register extensions, create rules, and create data elements to send data from your mobile app to Adobe Experience Cloud solutions."

Reply:
"For the push notifications data flow, the primary configurations required in Launch are:"